ldms/scripts/ldms-run-static-tests.sh.in

```diff
@@ -0,0 +1,96 @@
+#! /bin/sh
```

Comment:
How about moving this script to ldms/src/test somewhere instead of ldms scripts?